__tests__/integration/app.test.js

```diff
@@ -179,6 +177,67 @@ describe("GET /api/articles", () => {
 	});
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+	test("200: responds with an array of comments for the given article_id with correct properties", () => {
+		return request(app)
+			.get("/api/articles/3/comments")
+			.expect(200)
+			.then(({ body }) => {
+				const { comments } = body;
+				expect(comments).toBeInstanceOf(Array);
```

Comment:
This test is checking that comments is an array, but the following two tests of checking the length and using a forEach also indirectly test that comments is an array, which makes this assertion a little redundant.